### PR TITLE
New/password reset

### PR DESCRIPTION
**Pull request by olyakolesnichenko**

### CODE REVIEW

File: client/src/scenes/Login/index.jsx [hunk 1]
```diff
@@ -69,30 +71,39 @@ class Login extends Component {
   render() {
       const { isLoading, isEmailValid, isPasswordValid } = this.state;
       return !this.props.isAuthorized ? (
-          <Grid textAlign="center" verticalAlign="middle" className="fill">
-              <Grid.Column style={{ maxWidth: 450 }}>
-                  <Header as="h2" color="blue" textAlign="center">
+          <Grid textAlign="center" style={{ height: '100vh' }} verticalAlign="middle" className="login-grid">
```
Comment (alexandrtovmach) [inline client/src/scenes/Login/index.jsx]: Are you sure that's no other way to set height?

File: client/src/scenes/Login/index.jsx [hunk 2]
```diff
@@ -69,30 +71,39 @@ class Login extends Component {
...
+              <Grid.Column style={{ maxWidth: 400 }}>
```
Comment (alexandrtovmach) [inline client/src/scenes/Login/index.jsx]: Are you sure that's no other way to set max-width?

File: client/src/scenes/Reset/index.jsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  render() {
+      const { isLoading, isPasswordValid, isRepeatPasswordValid, isConfirmedPassword } = this.state;
+      const { isAuthorized, message, passwordNotReset, passwordReset } = this.props;
+      const succsessMessage = passwordReset ? <Message color="teal">{message}</Message> : null;
+      const failureMessage = passwordNotReset ? <Message color="red">{message}</Message> : null;
+      this.renderRedirect(succsessMessage);
+      if (this.state.redirect === true) return <Redirect to="/login" />;
+      return !isAuthorized ? (
+          <Grid textAlign="center" style={{ height: '100vh' }} verticalAlign="middle" className="reset-grid fill">
+              <Grid.Column style={{ maxWidth: 400 }}>
```
Comment (alexandrtovmach) [inline client/src/scenes/Reset/index.jsx]: Same, inline styles

File: server/api/services/user.service.js [hunk 4]
```diff
@@ -14,7 +18,41 @@ const checkUsernameExists = async ({ username }) => {
...
+  };
+};
+
+const sendForgetPasswordEmail = async ({ email }) => {
+  const isExist = await checkEmailExists({ email });
+  if (!isExist.emailExists) {
+    return { failure: 'Email is not exist' };
+  }
+  const user = await UserRepository.getByEmail(email);
+  const token = jwt.sign({ data: user.email }, secret, { expiresIn: '1h' });
```
Comment (alexandrtovmach) [inline server/api/services/user.service.js]: Should be used `server/helpers/token.helper.js`

File: server/helpers/email.helper.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
+const AWS = require('aws-sdk');
+
+const accessKey = process.env.AWS_SES_ACCESS_KEY;
+const secretKey = process.env.AWS_SES_SECRET_KEY;
+const awsRegion = process.env.AWS_SES_REGION;
```
Comment (alexandrtovmach) [inline server/helpers/email.helper.js]: Need to create `server/config/aws.config.js`

File: server/server.js [hunk 6]
```diff
@@ -11,6 +11,7 @@ const errorHandlerMiddleware = require('./api/middlewares/error-handler.middlewa
 const app = express();
 app.use(cors());
 
+app.use(cors());
```
Comment (alexandrtovmach) [inline server/server.js]: Duplicate

File: client/src/scenes/Login/index.jsx (hunk 1 above)
Comment (olyakolesnichenko) [reply client/src/scenes/Login/index.jsx]: бездумно взяли из примера семантика
https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/src/layouts/LoginLayout.js

Comment (olyakolesnichenko) [reply client/src/scenes/Login/index.jsx]: fixed

File: client/src/scenes/Login/index.jsx (hunk 2 above)
Comment (olyakolesnichenko) [reply client/src/scenes/Login/index.jsx]: fixed

File: client/src/scenes/Reset/index.jsx (hunk 3 above)
Comment (olyakolesnichenko) [reply client/src/scenes/Reset/index.jsx]: fixed

File: server/server.js (hunk 6 above)
Comment (olyakolesnichenko) [reply server/server.js]: fixed

File: server/helpers/email.helper.js (hunk 5 above)
Comment (andreoven) [reply server/helpers/email.helper.js]: fixed

File: server/api/services/user.service.js (hunk 4 above)
Comment (andreoven) [reply server/api/services/user.service.js]: fixed

**Pull request merged**